### PR TITLE
MDR Multicluster ODF external mode deployment with MDR operator configurations

### PR DESCRIPTION
**Pull request by shylesh**
This PR addresses:
1. External mode ODF deployment for MDR multicluster scenario
2. Deploy MDR operators

### CODE REVIEW

**Comment by shylesh** [comment]
Rebased on latest master

**Comment by shylesh** [comment]
Rebased and resolved conflicts

**Comment by shylesh** [comment]
Rebased on latest master

**Comment by shylesh** [comment]
Deployment passed here: https://url.corp.redhat.com/mdrdeployment

**Review by petr-balogh — DISMISSED** [review]

**Review by OdedViner — DISMISSED** [review]

**Review by OdedViner — APPROVED** [review]

File: ocs_ci/deployment/helpers/external_cluster_helpers.py [hunk 1]
```diff
@@ -537,11 +540,44 @@ def get_external_cluster_client():
...
         )
     nodes = config.EXTERNAL_MODE["external_cluster_node_roles"]
+    node_role = None
+    if config.multicluster:
+        # In case of multicluster we need to run the commands from
+        # admin node
+        node_role = "_admin"
+    else:
+        node_role = "client"
+
```
[the remark this answers is not in the document]
Comment (shylesh) [reply ocs_ci/deployment/helpers/external_cluster_helpers.py:545]: done

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**